### PR TITLE
GEOMESA-1348 Adding distributed runtime version to tools command

### PR DESCRIPTION
**Pull request by elahrvivaz**
Signed-off-by: Emilio Lahr-Vivaz <elahrvivaz@ccri.com>

### CODE REVIEW

File: geomesa-utils/pom.xml [hunk 1]
```diff
@@ -157,6 +157,17 @@
                 </includes>
             </resource>
         </resources>
+
+        <plugins>
+            <plugin>
+                <groupId>pl.project13.maven</groupId>
```
Comment (jnh5y) [inline geomesa-utils/pom.xml]: Does moving the git-commit-id-plugin to utils mean that all the fat jars end up with a geomesa.properties file?

File: geomesa-accumulo/geomesa-accumulo-tools/src/main/scala/org/locationtech/geomesa/tools/accumulo/commands/AccumuloVersionCommand.scala [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+      ds.getIteratorVersion
+    } catch {
+      case NonFatal(e) => "unavailable (use optional arguments to check distributed runtime version)"
+    } finally {
+      if (ds != null) {
+        ds.dispose()
+      }
+    }
+
+    println(s"GeoMesa version: $ProjectVersion")
```
Comment (jnh5y) [inline geomesa-accumulo/geomesa-accumulo-tools/src/main/scala/org/locationtech/geomesa/tools/accumulo/commands/AccumuloVersionCommand.scala]: Is there a way to convey that this version is the version of the GeoMesa tools?

File: geomesa-accumulo/geomesa-accumulo-tools/src/main/scala/org/locationtech/geomesa/tools/accumulo/AccumuloRunner.scala [hunk 3]
```diff
@@ -67,15 +68,25 @@ object AccumuloRunner extends Runner {
...
-      if (params.password == null) {
-        params.password = Prompt.readPassword()
+      if (p.instance == null) {
+        p.instance = try {
+          // this will hang for 60+ seconds if it's not configured - so we wrap in a future and only wait 1s
+          import scala.concurrent.duration._
+          Await.result(Future(HdfsZooInstance.getInstance().getInstanceName)(ExecutionContext.global),  1000.millis)
+        } catch {
+          case NonFatal(e) => logger.warn(s"Exception getting zoo instance: ${e.toString}"); null
+        }
```
Comment (jnh5y) [inline geomesa-accumulo/geomesa-accumulo-tools/src/main/scala/org/locationtech/geomesa/tools/accumulo/AccumuloRunner.scala]: Should this refactoring of tools options be backported to 1.2.x?

File: geomesa-utils/pom.xml (hunk 1 above)
Comment (elahrvivaz) [inline geomesa-utils/pom.xml]: it was already in geomesa-utils, so anything using that should get it. Previously it just had the pom version - this adds in the git values.

File: geomesa-accumulo/geomesa-accumulo-tools/src/main/scala/org/locationtech/geomesa/tools/accumulo/commands/AccumuloVersionCommand.scala (hunk 2 above)
Comment (elahrvivaz) [inline geomesa-accumulo/geomesa-accumulo-tools/src/main/scala/org/locationtech/geomesa/tools/accumulo/commands/AccumuloVersionCommand.scala]: "GeoMesa tool version" ?

File: geomesa-accumulo/geomesa-accumulo-tools/src/main/scala/org/locationtech/geomesa/tools/accumulo/AccumuloRunner.scala (hunk 3 above)
Comment (elahrvivaz) [inline geomesa-accumulo/geomesa-accumulo-tools/src/main/scala/org/locationtech/geomesa/tools/accumulo/AccumuloRunner.scala]: either way - what in general are we back-porting? i was thinking just bug fixes, but I can open another PR for 1.2 if you want.

File: geomesa-utils/pom.xml (hunk 1 above)
Comment (jnh5y) [inline geomesa-utils/pom.xml]: Ok, great.

File: geomesa-accumulo/geomesa-accumulo-tools/src/main/scala/org/locationtech/geomesa/tools/accumulo/commands/AccumuloVersionCommand.scala (hunk 2 above)
Comment (jnh5y) [inline geomesa-accumulo/geomesa-accumulo-tools/src/main/scala/org/locationtech/geomesa/tools/accumulo/commands/AccumuloVersionCommand.scala]: works for me

File: geomesa-accumulo/geomesa-accumulo-tools/src/main/scala/org/locationtech/geomesa/tools/accumulo/AccumuloRunner.scala (hunk 3 above)
Comment (jnh5y) [inline geomesa-accumulo/geomesa-accumulo-tools/src/main/scala/org/locationtech/geomesa/tools/accumulo/AccumuloRunner.scala]: Good question...

File: geomesa-accumulo/geomesa-accumulo-tools/src/main/scala/org/locationtech/geomesa/tools/accumulo/commands/AccumuloVersionCommand.scala (hunk 2 above)
Comment (elahrvivaz) [inline geomesa-accumulo/geomesa-accumulo-tools/src/main/scala/org/locationtech/geomesa/tools/accumulo/commands/AccumuloVersionCommand.scala]: updated

**Pull request merged**